packages/libs/client-utils/package.json

```diff
@@ -15,58 +15,52 @@
   ],
   "exports": {
     ".": {
-      "import": "./lib/index.js",
-      "require": "./lib/index.js",
-      "default": "./lib/index.js",
-      "types": "./lib/index.d.ts"
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js",
+      "types": "./lib/esm/index.d.ts"
```

Comment:
I have read somewhere that "types" should be first in this object to get it to work.

Reply:
interesting, I'll try it

Reply:
it works regardless of the "types" place, but maybe there is an edge case. I will move it to top, its harmless